project/src/test/scala/ScalastyleInspectionsGeneratorTest.scala

```diff
@@ -0,0 +1,141 @@
+        ),
+        Some("Scaladoc is generally considered a good thing. Within reason."),
+        WarningLevel,
+        List(
+          Param("ignoreRegex", StringType, "^$"),
+          Param("ignoreTokenTypes", StringType, "^$"),
+          Param("ignoreOverride", BooleanType, "false"),
+          Param("indentStyle", StringType, "anydoc")
+        )
+      )
+    )
+
+    val result = ScalastyleInspectionsGenerator.extractInspections(inspections, docs, conf)
+
+    result shouldBe expected
+  }
+
+  "transform" should "successfully transform the code template" in {
+    val template =
+      """
+        |object ScalastyleInspections { 
+        | val AllInspections: Seq[ScalastyleInspection] = ???
+        |}
+    """.stripMargin
+
+    val source = template.parse[Source].get
+
+    val inspections = List(
+      ScalastyleInspection(
+        "org.scalastyle.file.FileTabChecker",
+        "line.contains.tab",
+        "Line contains Tab",
+        "Check that there are no tabs in a file",
+        None,
+        Some("Some say that tabs are evil."),
+        WarningLevel,
+        List()
+      ),
+      ScalastyleInspection(
+        "org.scalastyle.file.FileLengthChecker",
+        "file.size.limit",
+        "File length",
+        "Check the number of lines in a file",
+        None,
+        Some("Files which are too long can be hard to read and understand."),
+        WarningLevel,
+        List(Param("maxFileLength", IntegerType, "1500"))
+      )
+    )
+
+    val expected =
+      """
+        |object ScalastyleInspections {
+        |val AllInspections: Seq[ScalastyleInspection] = List(
+        | ScalastyleInspection(
+        |   clazz = "org.scalastyle.file.FileTabChecker",
+        |   id = "line.contains.tab",
+        |   label = "Line contains Tab",
+        |   description = "Check that there are no tabs in a file",
+        |   extraDescription = None,
+        |   justification = Some("Some say that tabs are evil."),
+        |   defaultLevel = WarningLevel,
+        |   params = List()
+        | ),
+        | ScalastyleInspection(
+        |   clazz = "org.scalastyle.file.FileLengthChecker",
+        |   id = "file.size.limit",
+        |   label = "File length",
+        |   description = "Check the number of lines in a file",
+        |   extraDescription = None,
+        |   justification = Some("Files which are too long can be hard to read and understand."),
+        |   defaultLevel = WarningLevel,
+        |   params = List(Param(name = "maxFileLength", typ = IntegerType, default = "1500"))
+        | )
+        |)
+        |}
+      """.stripMargin
+
+    val result = ScalastyleInspectionsGenerator.transform(source, inspections)
+
+    result.structure == expected.parse[Source].get.structure shouldBe true
```

Comment:
I may be wrong, but it is not this redundant?
I mean `shoudlBe` should just call `==` if not especial Equality is present, right?

Reply:
you're right - it wasn't necessary, I've replaced it with just `shouldBe`